### PR TITLE
Bool framework update model names

### PR DESCRIPTION
**Pull request by jsmcibm**
Updates to documentation, and jupyter notebooks, and a config file, mostly connected with renaming models in HF repository.

### CODE REVIEW

**Review by franzmpub — APPROVED** [review]
LGTM

File: examples/boolqa/README.md [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
-python examples/mrc/run_mrc.py --model_name_or_path ${BOOLEAN_MODEL_NAME} \
+python examples/mrc/run_mrc.py --model_name_or_path ibm/tydiqa-primary-task-xlm-roberta-large \
        --output_dir ${OUTPUT_DIR} --fp16 \
        --per_device_eval_batch_size 128 --overwrite_output_dir \
        --do_boolean --boolean_config  examples/boolqa/tydi_boolqa_config.json
 ```
-or step-by-step.
 The option `--do_boolean` supercedes the `--do_eval` option, and runs the following four-stage process:
 
 - **M**achine **R**eading **C**omprehension: given a question and and answer, find a representative span that may contain a short answer. This is analyzed in detail in the tydiqa.ipynb
```
Comment (seirasto) [inline examples/boolqa/README.md:15]:
```suggestion
- **M**achine **R**eading **C**omprehension: given a question and answer, find a representative span that may contain a short answer. This is analyzed in detail in the tydiqa.ipynb

```

File: examples/boolqa/README.md [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
 - **Q**uestion **T**ype **C**lassification: given the question, decide if it is boolean or short_answer
-- **Ev**idence **C**lassification: given a question and a short answer span, decide the short answer span supports yes or no. 
+- Answer classification (or **Ev**idence **C**lassification): given a question and an answer span, decide whether the span supports yes or no. 
 - **S**core **N**ormalization - span scores may have different dynamic ranges according as whether the question is boolean or short_anwer. Normalize them uniformally to `[0,1]`.
```
Comment (seirasto) [inline examples/boolqa/README.md:18]:
```suggestion
- **S**core **N**ormalization - span scores may have different dynamic ranges according to whether the question is boolean or short_anwer. Normalize them uniformally to `[0,1]`.

```

File: notebooks/boolqa/evc.ipynb [hunk 3]
```diff
@@ -2,15 +2,15 @@
...
    "cell_type": "markdown",
-   "id": "f85995d8",
+   "id": "e139f661",
    "metadata": {},
    "source": [
-    "# Evidence classification for boolean questions\n",
+    "# Answer classification for boolean questions\n",
     "\n",
-    "In this notebook, we look at the evidence classifation, which is a component in the Tydi pipeline which decides whether a boolean question should be answered `yes` or `no`.\n",
+    "In this notebook, we look at the answer (evidence) classifation, which is a component in the TyDiQA pipeline which decides whether a boolean question should be answered `yes` or `no`, based on a passage selected by machine reading comprehension component.\n",
```
Comment (seirasto) [inline notebooks/boolqa/evc.ipynb:10]:
```suggestion
    "In this notebook, we look at the answer (evidence) classifation, which is a component in the TyDiQA pipeline which decides whether a boolean question should be answered `yes` or `no`, based on a passage selected by the machine reading comprehension component.\n",
```

**Review by seirasto — APPROVED** [review]
LGTM a few minor grammar suggestions

**Review by mihaela-bornea — APPROVED** [review]
LGTM

**Comment by jsmcibm** [comment]
update to documentation, Jupyter notebooks triggered by name changes in HF repo

**Pull request merged**